inc/saihostintf.h

```diff
@@ -186,7 +191,7 @@ typedef enum _sai_hostif_trap_type_t
     /** default action is forward */
     SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V3_REPORT = 0x000000009,
 
-    /** default action is trap */
```

Comment:
In the new spec, you have to explicitly create traps, to match the unified object model.
Before you create a trap object, the behavior chosen is 
Drop L2 PDUs, IP2ME, pipeline exceptions
Forward L3 protocols
Drop user defined traps CPU copy

This choice is done for clean behavior.
If application wants to receive traps, it needs to explicitly create them. And so is the case for sample packet - application just needs to create the trap for it